lib/ClangImporter/ImportName.h

```diff
@@ -250,7 +250,8 @@ class NameImporter {
 
   /// Determine the Swift name for a Clang decl
   ImportedName importName(const clang::NamedDecl *decl,
-                          ImportNameVersion version);
+                          ImportNameVersion version,
+              clang::DeclarationName PreferredName = clang::DeclarationName());
```

Comment:
Style nitpick: please just wrap all arguments to be double-indented rather than dedenting from the end.